todo_app/view_model.py

```diff
@@ -9,18 +13,26 @@ def wrapper_split_result_by_updated_time(self, list_category,
         # Avoid iteration over a huge list, if not needed with this check
         if from_threshold == datetime.min and to_threshold == datetime.max:
             final_list = intermediate_list 
+            log.debug(f'get_cards({list_category}) returns {len(final_list)} cards')
             return final_list
         else:
             for x in intermediate_list:
                 if from_threshold <= x.updated_time < to_threshold: 
                     final_list.append(x)
+            log.debug(
+                f'get_cards({list_category}) returns {len(final_list)} cards \
+                    which are between {from_threshold.strftime("%b %d %Y %H:%M:%S")} \
+                        and {to_threshold.strftime("%b %d %Y %H:%M:%S")}')
```

Comment:
Minor, but you'll notice if you look at your logs that this prints out with the indents which makes it look a little silly - there are [a few options for how you can avoid that discussed here](https://stackoverflow.com/questions/10660435/pythonic-way-to-create-a-long-multi-line-string) but possibly easiest would be concatenating;
```python
log.debug(
                f'get_cards({list_category}) returns {len(final_list)} cards' +
                ' which are between {from_threshold.strftime("%b %d %Y %H:%M:%S")}' +
                ' and {to_threshold.strftime("%b %d %Y %H:%M:%S")}'
            )
```

Reply:
This stumped me initially - but I went through a few comments and it explains! I have made appropriate changes.